src/cosalib/qemuvariants.py

```diff
@@ -91,6 +91,10 @@
             "--format=oldgnu"
         ]
     },
+    "kubevirt": {
+        "image_format": "qcow2",
```

Comment:
I had the impression that here we have the line where the base-image and the distribution mechanism are distinguished. Like also the aws image is built here as vmdk, and then pushed as ami later.

Reply:
Oh I don't mean having an FQIN here. I mean the file format itself would be an OCI archive. So then users could directly do e.g. `skopeo copy oci-archive:... docker://...`. We already do this today for the OSTree container though it's not an official user-facing artifact (notice how the "download" link for the OSTree commit in https://builds.coreos.fedoraproject.org/browser points to an `ociarchive` file).

Reply:
I would agree it makes sense to have the container oci-archive stored in S3, and then the upload process is just a copy and not actually making the image.

(This would also drop our dependency on google/go-containerregistry)

Reply:
Done.